.github/workflows/release.yml

```diff
@@ -74,10 +74,17 @@ jobs:
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v2
 
+    - name: Upload CodeQL Analysis Results
+      uses: actions/upload-artifact@v3
+      with:
+        name: azureauth-static-code-analysis-${{ matrix.runtime }}.sarif
+        path: results/csharp.sarif

```

Comment:
This is going to override the results from each job, with the last one winning won't it? Could we instead upload all 3 results from each platform job - similar to how you've used the runtime in the name for this step?

Reply:
Because we specified the argument `name`, there will be separate files like
- azureauth-static-code-analysis-windows-x64.sarif
- azureauth-static-code-analysis-osx-x64.sarif
- azureauth-static-code-analysis-osx-arm.sarif
And ideally it won't override our current artifacts.

Reply:
Oh I see the artifact is uploaded as the name attribute, not the path. Got it!